storagenode/piecestore/endpoint_test.go

```diff
@@ -33,7 +39,7 @@ func TestUploadAndPartialDownload(t *testing.T) {
 	_, err = rand.Read(expectedData)
 	require.NoError(t, err)
 
-	err = planet.Uplinks[0].Upload(ctx, planet.Satellites[0], "test/bucket", "test/path", expectedData)
+	err = planet.Uplinks[0].Upload(ctx, planet.Satellites[0], "testBucket", "test/path", expectedData)
```

Comment:
use lowercase, so it matches kaloyan-s PR

Reply:
done